code/modules/holiday/holidays.dm

```diff
@@ -312,11 +312,16 @@
 	begin_month = SEPTEMBER
 	drone_hat = /obj/item/clothing/head/pirate
 
-/datum/holiday/pirate/greet()
-	return "Ye be talkin' like a pirate today or else ye'r walkin' tha plank, matey!"
```

Comment:
You're DELETING THE PIRATE HOLIDAY FLOYD WHERES IT GOING

Reply:
YOURE STILL DELETING THE GREET

Reply:
WHAT DID PIRATE DAY DO TO YOU

Reply:
NOOOOOOOOOOOOOOOOOOOOO